sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/CassandraClusterCommandPublicParameters.cs

```diff
@@ -0,0 +1,130 @@
+        /// <param name="status"> Status of the command. </param>
+        /// <param name="outputFile"> The name of the file where the result is written. </param>
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        internal CassandraClusterCommandPublicParameters(string command, string commandId, BinaryData arguments, string host, bool? isAdmin, bool? cassandraStopStart, bool? readWrite, string result, CommandStatus? status, string outputFile, IDictionary<string, BinaryData> serializedAdditionalRawData)
+        {
+            Command = command;
+            CommandId = commandId;
+            Arguments = arguments;
+            Host = host;
+            IsAdmin = isAdmin;
+            CassandraStopStart = cassandraStopStart;
+            ReadWrite = readWrite;
+            Result = result;
+            Status = status;
+            OutputFile = outputFile;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> The command which should be run. </summary>
+        public string Command { get; }
+        /// <summary> The unique id of command. </summary>
+        public string CommandId { get; }
+        /// <summary>
+        /// The arguments for the command to be run
+        /// <para>
+        /// To assign an object to this property use <see cref="BinaryData.FromObjectAsJson{T}(T, System.Text.Json.JsonSerializerOptions?)"/>.
+        /// </para>
+        /// <para>
+        /// To assign an already formatted json string to this property use <see cref="BinaryData.FromString(string)"/>.
+        /// </para>
+        /// <para>
+        /// Examples:
+        /// <list type="bullet">
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson("foo")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("\"foo\"")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson(new { key = "value" })</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("{\"key\": \"value\"}")</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// </list>
+        /// </para>
+        /// </summary>
+        public BinaryData Arguments { get; }
+        /// <summary> IP address of the cassandra host to run the command on. </summary>
+        public string Host { get; }
+        /// <summary> Whether command has admin privileges. </summary>
+        public bool? IsAdmin { get; }
+        /// <summary> If true, stops cassandra before executing the command and then start it again. </summary>
+        public bool? CassandraStopStart { get; }
```

Comment:
Can we give a better name for this boolean property? like
```suggestion
        public bool? ShouldStopCassandraBeforeStart { get; }
```

Reply:
This is a property name used in our backend, maybe better to keep it.

Reply:
It's .NET SDK guideline: https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members

> DO name Boolean properties with an affirmative phrase (CanSeek instead of CantSeek). Optionally, you can also prefix Boolean properties with "Is", "Can", or "Has", but only where it adds value.

Reply:
You can use `x-ms-client-name` to rename: https://github.com/Azure/azure-sdk-for-net/blob/7a1e1f14d5d4c0f7089bb0f700c987a92d155224/sdk/dns/Azure.ResourceManager.Dns/src/autorest.md?plain=1#L117